### PR TITLE
*: Cleanup table codec

### PR DESCRIPTION
**Pull request by shenli**
Prepare for row-based storage model

### CODE REVIEW

File: tablecodec/tablecodec.go [hunk 1]
```diff
@@ -225,14 +241,26 @@ func EncodeTablePrefix(tableID int64) kv.Key {
...
 	buf = append(buf, tablePrefix...)
 	buf = codec.EncodeInt(buf, tableID)
 	buf = append(buf, recordPrefixSep...)
 	return buf
 }
 
-// Index prefix is "t[tableID]_i".
-func appendTableIndexPrefix(buf []byte, tableID int64) []byte {
+// GenTableIndexPrefix composes index prefix with tableID: "t[tableID]_i".
+func GenTableIndexPrefix(tableID int64) kv.Key {
```
Comment (coocood) [inline tablecodec/tablecodec.go]: I think `GenTableIndexPrefix` should call `appendTableIndex`.

**Comment by zimulala** [comment]
LGTM

File: tablecodec/tablecodec.go (hunk 1 above)
Comment (shenli) [inline tablecodec/tablecodec.go]: PTAL @coocood

**Comment by coocood** [comment]
LGTM

**Pull request merged**